src/components/pages/deploy/DeployForm.tsx

```diff
@@ -283,4 +284,8 @@ const component: React.FC = ({ children }) => {
   );
 };
 
-export const DeployForm: typeof component = memo(component);
+UDeployForm.propTypes = {
+  children: PropTypes.node.isRequired,
+};
```

Comment:
In that case I'd rather have it as explicit Props interface/type rather than using `propTypes`

Reply:
Yup! I figured this out later but didn't change it everywhere. Will change.